src/main/java/com/exasol/adapter/dialects/SqlDialectRegistry.java

```diff
@@ -1,7 +1,13 @@
 package com.exasol.adapter.dialects;
 
 import java.sql.Connection;
-import java.util.*;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.ServiceLoader;
+import java.util.Set;
```

Comment:
Do you do these imports on purpose? it's more than 3, so it's usually a star

Reply:
Fixed formatter settings.